_docs/setup/kubernetes/multicluster-install.md

```diff
@@ -0,0 +1,172 @@
+          serverAddress: "10.23.230.11"
+```
+
+A unique security context is required to describe how to securely access
+each Kubernetes cluster in the system.  In the above example,
+`falkor07.kube.conf` is the Istio control plane credential file. The
+`config.istio.io/pilotCfgStore: True` flag is set in the condition this
+`clusterregistry.k8s.io` descriptor describes the Istio control plane Kubernetes
+cluster.  For remotes, please set `pilotCfgStore: False`.
+
+**Important**: The implementation only uses the
+`config.istio.io/pilotCfgStore` and `config.istio.io/accessConfigFile`
+annotations, although every other annotation and spec is validated for
+correct syntax.  They may be set to dummy values, as long as they are
+syntactically correct.
+
+If the prerequisites are met, the credentials for each Kubernetes cluster
+will also be present in `$HOME/multicluster`.
+
+Assemble and create the configmap from these files in the Kubernetes cluster
+where the Istio control plane is operational:
+
+```bash
+kubectl create configmap -n istio-system multicluster -f $HOME/multicluster
+```
+
+## Deploy on the remote cluster
+
+Multicluster makes **one** Istio service mesh from all Kubernetes clusters
+that participate in multicluster.  Multicluster uses **one** Kubernetes
+cluster to run the Istio control plane.  **Any reasonable** number of remote
+Kubernetes clusters may be attached to the **one** Istio control plane.
+
+### Use kubectl with Helm to connect the remote cluster
+
+1. Use the helm template command on a remote to specify the Istio control plane:
+
+   ```bash
+   helm template install/kubernetes/helm/istio-remote --name istio-remote --set pilotEndpoint=`pod_ip_of_pilot_master` --set mixerEndpoint=`pod_ip_of_mixer_master` > $HOME/istio-remote.yaml
+    ```
+
+1. Instantiate the remote cluster's connection to the Istio control plane:
+
+   ```bash
+   kubectl create -f $HOME/istio-remote.yaml
+   ```
+
+### Alternatively use Helm and Tiller to connect the remote cluster
+
+1. If a service account has not already been installed for Helm, please
+install one:
+
+   ```bash
+   kubectl create -f install/kubernetes/helm/helm-service-account.yaml
+   ```
+
+1. Initialize Helm:
+
+   ```bash
+   helm init --service-account tiller
+   ```
+
+1. Install the Helm chart:
+
+   ```bash
+   helm install install/kubernetes/helm/istio-remote --name istio-remote --set pilotEndpoint=pod_ip_of_pilot_master --set mixerEndpoint=pod_ip_of_mixer_master`
+   ```
+
+### Mandatory Helm configuration parameters
+
+The `isito-remote` Helm chart requires the configuration of two specific variables defined in the following table:
+
+**Note** The `pilotEndpoint` and `mixerEndpoint` need to be resolvable via
+Kubernetes.
+
+| Helm Variable | Accepted Values | Default | Purpose of Value |
+| --- | --- | --- | --- |
+| `global.pilotEndpoint` | A valid IPv4 address | none | Specifies the Istio control plane's pilot Pod IP address |
```

Comment:
Is this really only ipv4?

Reply:
To answer your question, I think there is not really a good reason for that to be the case other than IPV6 just became a reality in 1.10 of Kubernetes (iiuc).  @pmichali is taking a look at adding an IPv6 implementation to Istio; detailed here are the steps he  went through to get that running: http://blog.michali.net/2018/03/06/istio-on-ipv6-kubernetes-undiscovered-country/.  I have not tested IPv6 and dualstack (ipv4+ipv6 in same Kubernetes system) is non-existent at the moment.  So its either IPv4, or IPv6, depending on how you set up your system.

To clarify, I will change this to be "A valid IP address".

Reply:
Istio - is not IPv6 capable at the moment so having Ipv4 here is factually correct,  but probably better to change this so there is one less place to fix to get IPv6 fully functional